packages/libs/error-stack/Cargo.toml

```diff
@@ -39,14 +38,14 @@ ansi-to-html = "0.1.2"
 once_cell = "1.17.0"
 supports-color = "2.0.0"
 supports-unicode = "1.0.2"
+owo-colors = "3"
```

Comment:
We can pin a specific version here

Reply:
fixed in [0d7afa6](https://github.com/hashintel/hash/pull/1980/commits/0d7afa6416abf3777b740df69a9b4e50cf358676)